docs/terminal/tassie.md

```diff
@@ -37,7 +37,14 @@ All aircraft should be kept on SIDs and STARs. If due to operational requirement
 ## YMLT
 Visual approaches are preferred into YMLT. If due to operational requirements, an aircraft is unable to accept a visual approach, coordination with **LT ADC** may be required.  
 
-Runway 32L is regularly the duty runway due to prevailing winds. To assist traffic flow in and out of the TCU, ATC will instruct aircraft to track for runway 32L via `IRSOM, NODAS, MLTSC` which keeps the aircraft within CTA and away from the departures stream.
+Runway 32L is regularly the duty runway due to prevailing winds. To assist traffic flow in and out of the TCU, ATC will instruct aircraft to track for runway 32L via one of the following routes:
+
+| Via              | Routing      |
+| ------------------ | --------------|
+| IRSOM  | NODAS MLTSC |
+| NUNPA  | ELREL MLTSA |
+
+These routes keep arrivals within CTA and away from the departure stream(s).
```

Comment:
just a note (maybe in one of the "!!! note" boxes) something along the lines of tracking aircraft via this routing **does not** constitute a voiceless coordination route, all aircraft still must be heads-up coordinated prior to 5 mins from boundary as per #Coordination

Reply:
Yeah easy, this ok?

![image](https://github.com/vatpac-technology/sops/assets/48381358/a47e5265-a756-4265-ad32-fbad101435c4)